### DISCUSSION TITLE
Reembolso e cancelamento de licença

### DISCUSSION THREAD

**Discussion by Elivelthon** [aclap-dev/video-downloadhelper#1382] [License & Purchase requests]
### What's your issue?

Olá! gostaria de cancelar e pedir um reembolso da minha compra de licença, pois o programa esta com muitos erros e não conseguir baixar os vídeos que preciso. 

### The 10 first digits (only 10) of your license number or email address

33d51e6383

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi, I sent you an email.
